test/arch_spec/arch_spec_example.yaml

```diff
@@ -19,7 +19,7 @@ cgra_defaults:
 
 tile_defaults:
   num_registers: 128
-  operations: ["add", "mul", "sub", "div", "rem", "fadd", "fmul", "fsub", "fdiv", "or", "not", "icmp", "fcmp", "sel", "cast", "sext", "zext", "shl", "vfmul", "fadd_fadd", "fmul_fadd", "data_mov", "ctrl_mov", "reserve", "grant_predicate", "grant_once", "grant_always", "loop_control", "phi", "constant"]  # comprehensive operation set
+  operations: ["add", "mul", "div", "fadd", "fmul", "fdiv", "logic", "cmp", "sel", "type_conv", "vfmul", "fadd_fadd", "fmul_fadd", "grant", "loop_control", "phi", "constant", "mem", "return", "mem_indexed", "alloca", "shift"]  # comprehensive operation set
```

Comment:
@n0thingNoob agree on this modification? Simulator needs to be updated.

CCing @Jackcuii for viz.